sky/serve/constants.py

```diff
@@ -53,7 +53,7 @@
 # do some log rotation.
 CONTROLLER_RESOURCES = {'cpus': '4+', 'disk_size': 200}
 
-# Due to the CPU/memory usage of the controller process launched with sky job (
+# Due to the CPU/memory usage of the controller process launched with sky jobs (
```

Comment:
accidental? feeling like we should keep the original comment to avoid get confused w/ managed jobs